JsonSchema.Tests/VocabularyTests.cs

```diff
@@ -136,7 +136,7 @@ public MaxDateKeyword(DateTime date)
 		}
 	}
 
-	public class MaxDateJsonConverter : JsonConverter<MaxDateKeyword>
+	private class MaxDateJsonConverter : JsonConverter<MaxDateKeyword>
```

Comment:
I don't mind everything in the tests being public.

Reply:
I put this back.